### PR TITLE
Adds support for Subscriptions

### PR DESCRIPTION
**Pull request by pratikkapasi**
Adds support for Subscriptions

### CODE REVIEW

File: src/Subscription.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        return parent::all($options);
+    }
+
+    public function cancel()
+    {
+        $relativeUrl = $this->getEntityUrl() . $this->id . '/cancel';
+
+        $data = [
+            'cancel_at_cycle_end' => 1
```
Comment (harman28) [inline src/Subscription.php]: @StSarc Is this the default behaviour we want in the SDK?

Comment (StSarc) [reply src/Subscription.php]: No. Default  behavior should be immediate

**Comment by harman28** [comment]
@pratikkapasi Modify verifyPaymentSignature to work with subscription responses as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ Changelog for Razorpay-PHP SDK. Follows [keepachangelog.com](http://keepachangel
 ## Unreleased
 ### Added
 - Support for VirtualAccount entity
+- Support for subscriptions
```
Comment (harman28) [inline CHANGELOG.md]: `Subscriptions`

**Pull request merged**